### PR TITLE
Add additional maven options

### PR DESCRIPTION
**Pull request by celanthe**
Added additional maven options as referenced in issue #9

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -55,7 +55,10 @@ runs:
     - name: Run maven
       run: |-
         test -z "${{ inputs.run-tests }}" && SKIP_TESTS="-DskipTests"
-        mvn -B package ${SKIP_TESTS}
+      mvn -B ${{ maven-additional-options:
```
Comment (gclaussn) [inline action.yml:58]: Are you sure that you can inline such an option? The option should be added at the top (line 3), where the other inputs are defined.

Additionally the three dots "...", where just a placeholder for the actual goals. So the mvn call at this place should look like this:
```
mvn -B ${{ maven-additional-options }} package ${SKIP_TESTS}
```

Comment (Langleu) [reply action.yml:58]: have to agree with @gclaussn .
The option itself is already defined in the file.
```
maven-release-options:
    description: Any extra Maven options for the release process (-DskipTests is already implied)
    required: false
    default: ""
```

I'd go as far as including the `inputs` -->
mvn -B ${{ inputs.maven-release-options }} package ${SKIP_TESTS}

File: action.yml [hunk 2]
```diff
@@ -55,7 +55,10 @@ runs:
...
+    description: Any extra Maven options
+    required: false
+    default: "" }} ...
```
Comment (lwille) [inline action.yml:61]: I think #9 specifially calls for extra options for the "non-release" invocation of maven.

However, this input can be defined just in the same way as `maven-release-options`.

```suggestion
      mvn -B ${{ inputs.maven-additional-options }} package ${SKIP_TESTS}
```

**Review by lwille — CHANGES_REQUESTED** [review]

**Comment by celanthe** [comment]
closing in favor of #13

**Pull request closed without merging**